src/main/java/org/reactome/release/qa/check/NewRegulationChecker.java

```diff
@@ -62,23 +68,57 @@ public QAReport executeQACheck() throws Exception
+    private boolean changedRegulatedByWithoutNewReviewed(GKInstance currentRlE, GKInstance previousRlE) throws Exception {
+
         // If previousRlE does not exist, we ignore it since that does not pertain to this particular QA check.
-        if (previousRlE != null) {
-            int currentRlERegulationCount = currentRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy).size();
-            int currentRlEReviewedCount = currentRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
-            int previousRlERegulationCount = previousRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy).size();
-            int previousRlEReviewedCount = previousRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
+        if (previousRlE == null) {
+            return false;
+        } else {
+            // Compare contents of the 'regulatedBy' attribute of the current and previous versions of the RlE.
+            boolean sameRegulatedBy = isSameRegulatedByValues(
+                    currentRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy),
+                    previousRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy)
+            );
+
+            int currentRlEReviewed = currentRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
+            int previousRlEReviewed = previousRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
 
             // The actual QA check
-            if (currentRlERegulationCount > previousRlERegulationCount && currentRlEReviewedCount == previousRlEReviewedCount) {
-                return true;
+            return !sameRegulatedBy && currentRlEReviewed == previousRlEReviewed;
+        }
+    }
+
+    /**
+     * Adapted from private InstanceUtilities.compareAttributes method. Takes the two 'regulatedBy' lists from the
+     * current and previous versions of a ReactionlikeEvent, and compares their contents, returning false if they differ.
+     * @param regulatedByInstancesCurrent List<GKInstance> -- Contents of currentRlE's 'regulatedBy' attribute
+     * @param regulatedByInstancesPrevious List<GKInstance> -- Contents of previousRlE's 'regulatedBy' attribute
+     * @return boolean -- true if lists are equal, false if not.
+     */
+    private boolean isSameRegulatedByValues(List<GKInstance> regulatedByInstancesCurrent, List<GKInstance> regulatedByInstancesPrevious) {
+        List<GKInstance> list1Copy = new ArrayList(regulatedByInstancesCurrent);
+        List<GKInstance> list2Copy = new ArrayList(regulatedByInstancesPrevious);
+        GKInstance instance1;
+        GKInstance instance2;
+        for (Iterator it = list1Copy.iterator(); it.hasNext();) {
+            instance1 = (GKInstance) it.next();
+            for (Iterator it1 = list2Copy.iterator(); it1.hasNext();) {
+                instance2 = (GKInstance) it1.next();
```

Comment:
I think you can put the declaration of `instance1` and `instance2` inline with their assignments on line 104 and 106